src/Native/System.Security.Cryptography.Native/evp.cpp

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include <stdint.h>
+#include <openssl/evp.h>
+
+#define SUCCESS 1
+
+/*
+Function:
+CreateEvpMdContext
+
+Direct shim to EVP_MD_CTX_create.
+*/
+extern "C" EVP_MD_CTX* CreateEvpMdContext()
+{
+    return EVP_MD_CTX_create();
+}
+
+/*
+Function:
+InitializeEvpDigest
+
+Direct shim to EVP_DigestInit_ex.
+*/
+extern "C" int32_t InitializeEvpDigest(EVP_MD_CTX* ctx, const EVP_MD* type, ENGINE* impl)
+{
+    return EVP_DigestInit_ex(ctx, type, impl);
```

Comment:
For 1:1 functions, we should just use our casing and otherwise leave the name alone: EvpDigestInitEx, etc.